src/components/layout/header/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+
+import { AppShell, Flex } from '@mantine/core';
+
+import SwitcherTheme from 'components/shared/switcherTheme/SwitcherTheme';
+
+interface Props {
+  opened: boolean;
+  toggle: () => void;
+}
+
+const Header: FC<Props> = () => {
+  return (
+    <AppShell.Header>
+      <Flex p="md" h="100%" w="100%" align="center" justify="space-between">
+        <span>Logo</span>
```

Comment:
what is `span` for?

Reply:
I put the text in a span to keep the Logo text from lying around without wrapping. This is a temporary solution until there is no logo.